service/client_routes.cc

```diff
@@ -72,3 +72,57 @@ future<std::vector<service::client_routes_service::client_route_entry>> service:
     }
     co_return result;
 }
+
+seastar::future<> service::client_routes_service::set_client_routes_inner(const std::vector<service::client_routes_service::client_route_entry>& route_entries) {
+    auto guard = co_await _group0_client.start_operation(_abort_source, service::raft_timeout{});
+    utils::chunked_vector<canonical_mutation> cmuts;
+
+    for (auto& entry : route_entries) {
+        auto mut = co_await make_update_client_route_mutation(guard.write_timestamp(), entry);
```

Comment:
Not related to the current version of this PR, but I wonder what timestamp would we pass here if we used `add_entry_unguarded`.